Adding events to calendar (#2713)
mainmatter/mainmatter.com: src/calendar/2025-05-28-PyCon2025.md

---
title: "PyCon Italia 2025"
image: "/assets/images/calendar/2025-05-28-PyCon2025/rust.png"
location: "Bologna"
url: https://2025.pycon.it/en/event/rust-python-interoperability-2
kind: workshop
color: purple
hero:
  image: "/assets/images/calendar/2025-05-28-PyCon2025/Bologna.png"
  imageAlt: "Evening falls on the warm, historic porticoes of Bologna, Italy, lit by golden streetlamps and steeped in charm."
---

> Our Rust expert Luca Palmieri will lead the Rust–Python Interoperability workshop, a hands-on session showing how to boost Python app performance by rewriting only the slow parts in Rust. Learn to work with pyo3, navigate the GIL, and build native extensions.

Reviewer: ```suggestion
Our Rust expert Luca Palmieri will host a Rust–Python Interoperability workshop, showing how to boost the performance of Python apps by rewriting only the slow parts in Rust. Learn to work with pyo3, navigate the GIL, and build native extensions.
```
